### PR TITLE
Update breaking change from strict mode class delcaration and class expr...

### PR DESCRIPTION
**Pull request by yuit**
...ession

### CODE REVIEW

File: Breaking-Changes.md [hunk 1]
```diff
@@ -119,4 +120,20 @@ Manually specify a type parameter
...
+ECMAScript 2015 Language Specification (ECMA-262 6<sup>th</sup> Edition) specifies that a ClassDeclaration and a ClassExpression are strict mode codes. 
+Thus, additional restriction will be applied when interpreting class declaration and class expression. 
+
+Examples:
+
+```ts
+class implements {}  // Invalid: implements is a reserved word in strict mode
+class C {
+	foo(arguments: any) {   // Invalid: "arguments" is not allow as a function argument
+		var eval = 10;      // Invalid: "eval" is not allowed as the left-hand-side expression
```
Comment (CyrusNajmabadi) [inline Breaking-Changes.md]: do you want 'var' here?  or in the next line?

Comment (yuit) [inline Breaking-Changes.md]: I want 'var' here and remove var in the next statement.

Comment (DanielRosenwasser) [inline Breaking-Changes.md]: These are tabs, please convert them to spaces.

File: Breaking-Changes.md [hunk 2]
```diff
@@ -119,4 +120,20 @@ Manually specify a type parameter
 ```ts
 var r9 = f10<any>('', () => (a => a.foo), 1);
 ```
- 
\ No newline at end of file
+
+#### Strict Mode Class Declaration and Class Expression
+ECMAScript 2015 Language Specification (ECMA-262 6<sup>th</sup> Edition) specifies that a ClassDeclaration and a ClassExpression are strict mode codes. 
+Thus, additional restriction will be applied when interpreting class declaration and class expression.
```
Comment (DanielRosenwasser) [inline Breaking-Changes.md]: %s/declaration/declarations/ and %s/expression/expressions/

File: Breaking-Changes.md [hunk 3]
```diff
@@ -119,4 +120,20 @@ Manually specify a type parameter
 ```ts
 var r9 = f10<any>('', () => (a => a.foo), 1);
 ```
- 
\ No newline at end of file
+
+#### Strict Mode Class Declaration and Class Expression
+ECMAScript 2015 Language Specification (ECMA-262 6<sup>th</sup> Edition) specifies that a ClassDeclaration and a ClassExpression are strict mode codes.
```
Comment (DanielRosenwasser) [inline Breaking-Changes.md]: Use *italics* for grammar productions.

File: Breaking-Changes.md [hunk 4]
```diff
@@ -119,4 +120,20 @@ Manually specify a type parameter
 ```ts
 var r9 = f10<any>('', () => (a => a.foo), 1);
 ```
- 
\ No newline at end of file
+
+#### Strict Mode Class Declaration and Class Expression
```
Comment (DanielRosenwasser) [inline Breaking-Changes.md]: Strict Mode Parsing for Classes Declarations and Class Expressions

File: Breaking-Changes.md (hunk 2 above)
Comment (yuit) [inline Breaking-Changes.md]: Not sure I understand what you mean

**Comment by DanielRosenwasser** [comment]
Can you resolve the merging issues?

**Pull request merged**